src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyGraph.cs

```diff
@@ -27,6 +27,13 @@ public sealed class DependencyGraph
             },
             new DependencyType
             {
+                Name = DependencyTypeName.UidInclude,
```

Comment:
Sound good!

---
In reply to: [96593710](https://github.com/dotnet/docfx/pull/1184#discussion_r96593710) [](ancestors = 96593710)